src/components/limited-restoration-panel/LimitedRestorationPanel.vue

```diff
@@ -1,25 +1,26 @@
 <template id="limited-restoration-panel">
   <v-radio-group class="mt-0 pt-0 ml-8" v-model="selectMonths" mandatory>
-    <v-radio id="twentyFour-radio" name="selectMonths" label="Two Years" :value=24 />
-    <v-radio id="eighteen-radio" name="selectMonths" label="18 months" :value=18 />
-    <v-radio id="twelve-radio" name="selectMonths" label="12 months" :value=12 />
-    <v-radio id="six-radio" name="selectMonths" label="6 months" :value=6 />
-    <v-row class="ml-0 mt-0">
+    <v-radio class="radio-button pt-2" id="twentyFour-radio" name="selectMonths" label="2 years" :value=24 />
+    <v-radio class="radio-button pt-2" id="eighteen-radio" name="selectMonths" label="18 months" :value=18 />
+    <v-radio class="radio-button pt-2" id="twelve-radio" name="selectMonths" label="12 months" :value=12 />
+    <v-radio class="radio-button pt-2" id="six-radio" name="selectMonths" label="6 months" :value=6 />
+    <v-row class="ml-0 mt-0 radio-button pt-2">
       <v-radio id="custom-months" name="selectMonths" value="customMonths" />
-      <v-text-field
-        class="shrink"
-        hide-details="auto"
+      <v-form ref="monthsRef">
+        <v-text-field
+        id="months-text-field"
```

Comment:
Please indent these lines. Leave the `/>` aligned with the `<v-text-field`.

Ideally our IDEs (or linter) would do this automatically. Anyone care to look into a "prettier" extension?